poc/frost.sage

```diff
@@ -234,7 +233,8 @@ for (name, G, H) in ciphersuites:
     config["hash"] = H.name
 
     # Create all inputs, including the group key and individual signer key shares
-    signer_keys, group_secret_key, group_public_key = trusted_dealer_keygen(G, MAX_SIGNERS, THRESHOLD_LIMIT)
+    group_secret_key = G.random_scalar() # XXX(chk) This might need to be a test vector
```

Comment:
Assigning it to this variable does spit out the group secret as a test vector correctly